test/test_core.py

```diff
@@ -5832,6 +5832,10 @@ def test_fs_write(self):
   def test_fs_emptyPath(self):
     self.do_run_in_out_file_test('fs/test_emptyPath.c')
 
+  @also_with_noderawfs
```

Comment:
Oh I forget to say we should me marking these as @crossplatform so that they run on windows.